packages/cli/src/commands/tablegen.ts

```diff
@@ -20,7 +20,7 @@ const commandModule: CommandModule<Options, Options> = {
   },
 
   async handler({ configPath }) {
-    const config = await loadStoreConfig(configPath);
+    const config = (await loadConfig(configPath)) as StoreConfig;
```

Comment:
no, we don't. Since the config is typescript, if you don't have the plugin, the config file will have the errors itself so validating it again doesn't seem necessary

Reply:
to clarify, we do get zod validation and the same zod errors, but in `mudConfig`/`mudCoreConfig` themselves, not in `tablegen`. When running `mud tablegen` the output of errors should remain unchanged. Things get kinda weird if you try to use a broken config in a frontend though, but you shouldn't do that

Reply:
got it. I think in order to get strong types on imports of the config we might have to move the zod validation somewhere else, but it's fine to think about that in a follow up

Reply:
you already get strong types from `mudConfig` actually, I added that as a little extra here. It works fine with zod, except for the quirks I mentioned if config is invalid